selenium/src/main/java/com/deque/html/axecore/extensions/WebDriverExtensions.java

```diff
@@ -121,11 +120,7 @@ public static String openAboutBlank(final WebDriver webDriver) {
     String currentWindow = webDriver.getWindowHandle();
 
     try {
-      JavascriptExecutor driver = (JavascriptExecutor) webDriver;
-      driver.executeScript("window.open('about:blank', '_blank')");
-      ArrayList<String> handles = new ArrayList<String>(webDriver.getWindowHandles());
-      String abHandle = handles.get(handles.size() - 1);
-      webDriver.switchTo().window(abHandle);

```

Comment:
I tried to look at the git history to see why we did it this way vs `newWindow(..)` but couldn't find anything